modules/heat_conduction/doc/content/modules/heat_conduction/nafems_t3_verif.md

```diff
@@ -0,0 +1,87 @@
+# NAFEMS Benchmark 1D Transient Heat Conduction T3
+
+## Problem Description
+
+A NAFEMS one-dimensional transient heat conduction problem
+[!citep](nafems_std_bench) was created and run
+using the heat conduction module in MOOSE. A description of the problem and a
+summary of the results follows.
+
+### Model
+
+A bar of length 0.1 meters and cross section of 0.01 meters in width and depth
+is modeled using uniform sized elements with either a coarse mesh (5 elements,
+[fig:t3_coarse_mesh]) or a fine mesh (10 elements, [fig:t3_fine_mesh]). A
+transient heat conduction simulation is performed with a total simulation time
+of 32 seconds.
```

Comment:
The fact that you did this in 1D, 2D, and 3D deserves a mention here too.

Reply:
Done. Pushing the changes now.